index.html

```diff
@@ -445,8 +459,8 @@
         If a developer wants to check multiple permissions at once, the editors
         recommend them to use Promises.all(). It should yield to the same
         result and allow this API to stay simple. If it happens to be a very
-        common use case, it should be easy to extend Permissions.query() to
-        accept a sequence&lt;&gt; too.
+        common use case, it should be easy to extend
+        <code>permissions.query()</code> to accept a sequence&lt;&gt; too.
```

Comment:
I don't think we should set this expectation. I think Promise.all is a great solution to this.